.gitignore

```diff
@@ -0,0 +1,12 @@
+build/
```

Comment:
Every repo should have a `.gitignore` at least covering the defaults for that language.